dcos/http.py

```diff
@@ -162,11 +174,23 @@ def request(method,
     if 'headers' not in kwargs:
         kwargs['headers'] = {'Accept': 'application/json'}
 
-    response = _request(method, url, is_success, timeout, **kwargs)
+    if verify is None and os.environ.get(constants.DCOS_SSL_VERIFY_ENV):
+        verify = os.environ.get(constants.DCOS_SSL_VERIFY_ENV)
```

Comment:
you already checked it about so you want: `os.environ[constants.DCOS_SSL_VERIFY_ENV]`